src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -196,6 +196,24 @@ public function getBarOffset(): int
         return floor($this->max ? $this->percent * $this->barWidth : (null === $this->redrawFreq ? min(5, $this->barWidth / 15) * $this->writeCount : $this->step) % $this->barWidth);
     }
 
+    public function getEstimated(): int
```

Comment:
`round` returns a float, not an int, do we need to cast the result?

Reply:
`floor` also returns a float, but `getBarOffset` also has an `int` return type. Since strict_types isn't declared, this poses no problems?

Reply:
What about fixing the inconsistency? By either returning a float or casting to a string.

Reply:
In that case I'd vote for returning a float. That would also increase accuracy for extremely large numbers in the case of time estimations.

Reply:
That works for me.

Reply:
Done